### PR TITLE
Checkin of unit test 8

### PR DESCRIPTION
**Pull request by subutai**

### CODE REVIEW

**Comment by subutai** [comment]
@BoltzmannBrain This an initial checkin for the word order test (RES-203). I've verified that a simple union pooler with a history buffer of 2 does quite well.  I have not tried TM yet.

File: projects/nlp/junit_test8.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+    docStr = unicode(document, errors='ignore')[0:100]
+    printTemplate.add_row( [docId, docStr, score, truePositives])
+
+  lengthOfTest = float(len(testData))
+
+  print printTemplate
+  print
+  print "Total score / avg score across all test documents:",
+  print summedScore, summedScore/lengthOfTest
```
Comment (BoltzmannBrain) [inline projects/nlp/junit_test8.py]: Less confusing would be 
```
print "Scores across all test documents"
print "  Total:", summedScore
print "  Avg:", summedScore/float(lengthOfTest)

**Comment by BoltzmannBrain** [comment]
@subutai I left one nitpick. And why test 8, isn't this the sixth?
Otherwise 👍

**Comment by subutai** [comment]
> And why test 8, isn't this the sixth

It's the sixth one we've implemented, but in the document listing all tests it's the 8th one.

**Pull request merged**